deployment-examples/aws/k8s/create_aws_cluster.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+
+## edit these environment variables
+## ----------------------
+export AWS_REGION_CODE="us-west-2"
```

Comment:
Here you have `AWS_REGION_CODE` and `AWS_CLUSTER_NAME` as fixed texts. In [deploy_to_eks.sh](https://github.com/commercetools/commercetools-subscriptions/pull/48/files#diff-2ab7b61674d59ff1e35679d77b0646d0f3ed6745b3301100203b6ebec89e2dba) you have env vars taken from somewhere as variables. Please unite the style and update the doc accordingly.

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/10191179ef546c4356d12fa916e19262eb5227cf